app/models/spell.rb

```diff
@@ -1,2 +1,21 @@
 class Spell < ApplicationRecord
+  SCHOOLS = [
```

Comment:
Is that a better practice for the list of schools? It's not like any more will be added (in the near future).